File: Nível 1/1066 - Pares, Ímpares, Positivos e Negativos/main.py
par = 0
impar = 0
positivo = 0
negativo = 0

for i in range(5):
    valor = int(input("A: "))

    if valor > 0:
        positivo += 1 
    if valor < 0:
        negativo += 1

    calculo = valor % 2 

    if calculo == 0:
        par += 1 
    if calculo != 0:
        impar += 1

print(f"{par} valor(es) par(es)")
print(f"{impar} valor(es) impar(es)")
print(f"{positivo} valor(es) positivo(s)")
print(f"{negativo} valor(es) negativo(s)")

File: Nível 1/1066 - Pares, Ímpares, Positivos e Negativos/main.lua
par = 0
impar = 0
positivo = 0
negativo = 0

for i = 1, 5 do
    valor = tonumber(io.read())

    if valor > 0 then
        positivo = positivo + 1
    end

    if valor < 0 then 
        negativo = negativo + 1
    end

    calc = valor % 2

    if calc == 0 then
        par = par + 1
    end

    if calc ~= 0 then
        impar = impar + 1
    end
end

print(string.format("%d valor(es) par(es)", par))
print(string.format("%d valor(es) impar(es)", impar))
print(string.format("%d valor(es) positivo(s)", positivo))
print(string.format("%d valor(es) negativo(s)", negativo))